lib/route-data.js

```diff
@@ -28,6 +29,8 @@ class RouteData {
     /** @type {!Element|undefined} */
     this.element = undefined;
     this.requiresAuthentication = requiresAuthentication !== false;
+
+    this.importFunction = importFunction;
```

Comment:
It might make sense to fallback to an empty function here, so the mixin later doesn't have to do as much checking before calling it. I could go either way though.